jupyterhub/notebook-images/overlays/additional/generic-data-science-notebook-imagestream.yaml

```diff
@@ -5,7 +5,8 @@ metadata:
     opendatahub.io/notebook-image: "true"
   annotations:
     opendatahub.io/notebook-image-url: "https://github.com/thoth-station/s2i-generic-data-science-notebook"
-    opendatahub.io/notebook-image-name: "Generic Data Scient Notebook Image"
+    opendatahub.io/notebook-image-name: "Standard Data Science"
+    opendatahub.io/notebook-python-dependencies: [{"name":"Boto3","version":"1.17.11"},{"name":"Kafka-Python","version":"2.0.2"},{"name":"Matplotlib","version":"3.1.3"},{"name":"Numpy","version":"1.20.2"},{"name":"Pandas","version":"1.2.3"},{"name":"Scipy","version":"1.6.2"}]
```

Comment:
Looking into this, I think this annotation needs to be on the `tags` level - the reason is that if we need to provide multiple versions of DS notebook, we need to make sure we can provide info for each of them - and they'd probably live in different tags - WDYT?

`spec.tags.annotations`:

* `opendatahub.io/notebook-python-dependencies`
* `opendatahub.io/notebook-tag-name`

I can imagine others like `cuda-compatibility` `verified-gpus` etc..

Reply:
i agree, we can have different versions.

Reply:
@harshad16 Do you define somewhere what software and versions are in the notebook image as well as the python dependancies? I think that is what @jeff-philips-18 is looking for in this case.

Reply:
i have updated it now